src/app/core/settings/settings.component.html

```diff
@@ -43,6 +43,28 @@
 						Black (AMOLED)
 					</label>
 				</div>
+          <div>
```

Comment:
Let's add an `h2` header here before these elements. Something like 'Change Font' should be okay